src/yb/common/partition.cc

```diff
@@ -900,7 +900,7 @@ string PartitionSchema::PartitionDebugString(const Partition& partition,
     } else {
```

Comment:
Don't forget to change `s.append("), (")` above to `s.append(", ")`.

Reply:
May be I missed this, sorry about that @jaki , fixed now